R/moduleInstall.R

```diff
@@ -361,15 +364,648 @@ setupRenv <- function(moduleLibrary, modulePkg) {
 
   Sys.setenv("RENV_PATHS_LIBRARY" = moduleLibrary)
 
-  print("Using the following paths:")
+  cat("Using the following paths:\n")
```

Comment:
Now we can't see this in the log anymore when running a dynamic module install right?

Why would we want that?

Reply:
This should be visible in both? This code is, for now, intentionally not used when installing a dynamic module, so I'm not 100% sure though.

Reply:
But I assume that later we will reuse this code also for dynamic modules right?

Those are going to be exactly the same after all, as the ones in jasp-desktop.

But im not sure what the gain is from changing them from `print` anyway?